cypress/fixtures/user.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Avril Douglas Campbell",
+  "address": "850 Argyle Street\nPort Alberni\nBritish Columbia\nV9Y 1V8",
+  "maritalStatus": "Married",
+  "children": "0",
+  "income": "$25,000"
```

Comment:
Should we change this in this PR to match the new T4 flow?

Reply:
Yeah, I can update it. I think let's wait for #28 to make it in.